api/node-echoprint-server/controllers/api.js

```diff
@@ -100,3 +115,73 @@ exports.ingest = function(req, res) {
     });
   });
 };
+
+/**
+* Upload
+*
+* Validates and uploads files
+*
+* @param {Object} req
+* @param {Object} res
+*/
+function upload(req,res) {
```

Comment:
The thunder that you will experience, would be brought to you by 2 angry people!